src/components/RegisterForm.vue

```diff
@@ -0,0 +1,306 @@
+<template>
+	<form @submit.prevent="handleSubmit(!v$.$invalid)" class="p-fluid">
+		<div class="field mb-4">
+			<div class="p-float-label">
+				<InputText
+					id="firstname"
+					v-model="v$.firstname.$model"
+					:class="{
```

Comment:
to można do jednej linijki zwinąć, jak używasz Prettiera albo czegos do automatycznego pilnowania styli to chyba srednio jest skonfigurowany